contracts/BucketAuction.sol

```diff
@@ -24,7 +27,10 @@ contract BucketAuction is IBucketAuction, ERC721M {
         uint256 maxMintableSupply,
         uint256 globalWalletLimit,
         address cosigner,
-        uint256 minimumContributionInWei
+        uint256 minimumContributionInWei,
+        uint256 startTimeUnixSeconds,
```

Comment:
these timestamps can be u64 to save a storage slot

Reply:
why were we using u256 for _activeStage? Was it intentional?